src/js/steps/letters/generateHtml.js

```diff
@@ -11,15 +11,19 @@ var $ = require('jquery');
 module.exports = function (options) {
   var defer = $.Deferred();
   return function (data) {
+    if (!data.name) {
+      return data;
+    }
+
     var $lettersContainer = $('<div />');
     $lettersContainer.attr({
       id: 'letters-container',
-      'class': 'aligned-center row leaded md-mar-b', // @todo: remove leaded
```

Comment:
@barneyfoxuk The todo has been removed, but `leaded` still remains in the class list